### ISSUE TITLE
Type error when using --timeout option

### ISSUE THREAD

**Issue by AssetKid** [raineorshine/npm-check-updates#1295]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 14.14`

---

## Steps to Reproduce

When using the `--timeout` option, there is a type error.

 **Sample Command**

```
ncu -u -i --timeout 99999 --concurrency 1 --target  minor
```

## Current Behavior

```
TypeError [ERR_INVALID_ARG_TYPE]: The "timeout" argument must be of type number. Received type string ('99999')
    at new NodeError (node:internal/errors:399:5)
    at validateNumber (node:internal/validators:178:11)
    at getTimerDuration (node:internal/timers:394:3)
    at new ClientRequest (node:_http_client:204:20)
    at request (node:https:360:10)
```

The same doesn't seem to happen for `concurrency`.

## Expected Behavior

The command should run without any errors.

**Comment by raineorshine** [comment]
Hi, thanks for reporting.

This must be related to a specific node or dependency version, as I am not able to reproduce this on my end.

Luckily, it's pretty obvious *what* is causing the problem, even if we don't know exactly *where*. I added a `parseInt` and published to `v16.10.9`. Please try it out at your convenience.

**Comment by AssetKid** [comment]
Thank you for the quick fix. It is working now.
